src/js/libgeo/GeoOps.js

```diff
@@ -371,6 +444,22 @@ geoOps.Free.updatePosition = function(el) {
     putStateComplexVector(param); // copy param
     el.homog = General.withUsage(param, "Point");
 };
+geoOps.Free.getRandomMove = function(el) {
+    var rcomplex = function() {
+        var fact = 0.1;
+        // debug only real remove later!
+        return CSNumber.complex((Math.random() - 0.5) * fact, (Math.random() - 0.5) * 0);
+    };
+    var oldpos = List.normalizeMax(el.homog);
+    var rvect = List.turnIntoCSList([rcomplex(), rcomplex(), CSNumber.real(0)]);
+    var move = List.scalmult(oldpos.value[2], rvect);
```

Comment:
As discussed, this kind of movement will preserve points at infinity. What's even worse, since you scale by `oldpos.z`, points at infinity won't move at all. That can easily prevent us from disproving some spurious conjectures.

The whole setup puts a bit too much emphasis on the choice of coordinates, for my taste. A user using a coordinate system with thousand units per pixel may experience different effects compared to one using a thousand pixels per unit. Since I'm still advocating large movements, it's the former case that has me worried: Could it happen that at a very low zoom setting, some configuration just doesn't move enough to actually disprove invalid conjectures? Can we estimate the chances of this, for some simple conjectures (i.e. without too much of an artificial dampening construction)? I don't see this aspect as a showstopper, but a personal concern nonetheless.